metrics/graphite.go

```diff
@@ -67,6 +67,8 @@ func graphite(c *GraphiteConfig) error {
 		switch metric := i.(type) {
 		case Counter:
 			fmt.Fprintf(w, "%s.%s.count %d %d\n", c.Prefix, name, metric.Count(), now)
+		case CounterFloat64:
+			fmt.Fprintf(w, "%s.%s.value %f %d\n", c.Prefix, name, metric.Value(), now)
```

Comment:
Well, as I mentioned above, I decided to rename the value field to `Value`.  Are you guys ok with that?

Reply:
I think it should be called `count` in exported metrics because it's a counter. The internal method name can be `Value` or `Count`, it doesn't matter that much to me.

Reply:
I could always change it back.

Reply:
@fjl Hey, I hope you don't think I was beig a jerk when I said "I could always change it back" right after you explicitly said that is what I should do.  Maybe it came across that way.  What happened was, your comment of "It should be called count ..." hadn't come up on my screen yet for some reason, so I was just kind of talking into the void.  When I finally saw your comment, I was like "Oh oh.  My comment probably didn't come across very well!"